src/GraphQLParser.Tests/Visitors/CountVisitorTests.cs

```diff
@@ -38,7 +38,7 @@ public async Task CountVisitor_Should_Count_Nodes(string text, int expectedCount
 
         var document = text.Parse();
 
-        await visitor.VisitAsync(document, context).ConfigureAwait(false);
+        await visitor.VisitAsync(document, context);
```

Comment:
https://xunit.net/xunit.analyzers/rules/xUnit1030

Reply:
Please note that we had to add ConfigureAwait calls to DataLoader tests to make them work properly. If we make similar changes in the main project we will need to be sure multithreaded tests work reliably.

Reply:
I understand. It's a matter of balance. For these repo I think `.ConfigureAwait(false)` calls in tests are just noise.